source/includes/_tipser-elements.md

```diff
@@ -168,38 +167,39 @@ export interface TipserProductModel {
     freeReturn: boolean;
 }
 ```
+When onAddToCart is being dispatched the handlers are triggered with object with cart size of type string and product of type TipserProductModel.
+<br><br><br><br><br><br><br><br><br><br><br><br>
```

Comment:
That's so ugly :(. There is really no other way than doing so many `br`s?

Reply:
I know, but I didn't want to make it a new "chapter" and it would be the only way to match the length of "content" part with the "code" part on the right.
This is the part:
![image](https://user-images.githubusercontent.com/12189762/59087864-36942b80-8906-11e9-9f14-c6f8561fc3f8.png)

Reply:
🤔

Reply:
If this is really the only way it can be done, then what do to do ?